tests/test_plugin.py

```diff
@@ -791,6 +791,7 @@ def test_rpc_command_hook(node_factory):
     l1.rpc.plugin_stop('rpc_command.py')
 
 
+@flaky
```

Comment:
Why is this now @flaky?

Reply:
Because it started failing randomly on Travis but I was not able to reproduce locally..

Btw, i can no longer restart Travis jobs.. Don't know if it's a perm error or anything